roles/kubernetes-apps/ansible/tasks/main.yaml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Kubernetes Apps | Wait for kube-apiserver
```

Comment:
There is no kubernetes-apps/tasks dir. I could make one, but I don't think it's a great idea a this time.